### PR TITLE
SpliceAI annotations

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality: SpliceAI annotations from VCF info if provided - as by e.g. MIPv9.

**How to test**:
1. (re)load a case with SpliceAI variants
1. note that these are being parsed correctly, and displayed on the variant page

Screenshot 2020-09-23 at 14 57 44

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by CR
- [ ] tests executed by

### CODE REVIEW

File: scout/server/blueprints/variant/templates/variant/variant_details.html [hunk 1]
```diff
@@ -182,6 +182,10 @@
           SPIDEX
           <span class="float-right">{{ variant.spidex_human }}</span>
         </li>
+        <li class="list-group-item">
+          SpliceAI DS Max
+          <span class="float-right">{{ variant.spliceai_ds_max }}</span>
```
Comment (northwestwitch) [inline scout/server/blueprints/variant/templates/variant/variant_details.html:187]: We're saving the variant spliceai string (bla|bla|bla|..) but we're not showing it on the variant page. Wouldn't users be interested in those values?

File: scout/server/blueprints/variant/templates/variant/variant_details.html [hunk 2]
```diff
@@ -182,6 +182,10 @@
           SPIDEX
           <span class="float-right">{{ variant.spidex_human }}</span>
         </li>
+        <li class="list-group-item">
```
Comment (northwestwitch) [inline scout/server/blueprints/variant/templates/variant/variant_details.html:185]: Perhaps we could have a condition here, like {% if variant.spliceai_ds_max %} so the li is not shown if there's no value in the field? But perhaps it's informative knowing that there is no spliceai. In that case you could write later in the span:
<span class="float-right">{{ variant.spliceai_ds_max or "-"}} </span>

**Review by northwestwitch — APPROVED** [review]
Looks perfect! I've got a couple of suggestion, see comments!

File: scout/server/blueprints/variant/templates/variant/variant_details.html (hunk 2 above)
Comment (dnil) [reply scout/server/blueprints/variant/templates/variant/variant_details.html:185]: Sure! I'll do the same for spidex!

File: scout/server/blueprints/variant/templates/variant/variant_details.html (hunk 1 above)
Comment (dnil) [reply scout/server/blueprints/variant/templates/variant/variant_details.html:187]: Do you know what values in particular? But, presumably I would be interested in all. Perhaps a hover? I'll try something!

Comment (northwestwitch) [reply scout/server/blueprints/variant/templates/variant/variant_details.html:187]: A hover would be perfect!

File: scout/server/blueprints/variant/templates/variant/variant_details.html (hunk 2 above)
Comment (dnil) [reply scout/server/blueprints/variant/templates/variant/variant_details.html:185]: It turns out spidex has a solution where the value is always looked up, and spidex_human set to "not_reported" when not found. To each his own I guess.. Chalk one for refactor. I currently like your solution better so I'll use it.

File: scout/server/blueprints/variant/templates/variant/variant_details.html (hunk 1 above)
Comment (dnil) [reply scout/server/blueprints/variant/templates/variant/variant_details.html]: Hm, I wonder if they will get that? I would probably have to look up the VCF-info key and then read the SpliceAI-paper to understand, but at least now it is displayed in some form.

**Comment by dnil** [comment]
Testing with real data is king. Turns out SpliceAI and SpliceAI_DS_max are in fact lists, not just floats: one value per gene, but not per transcript. 👀 

Anyway, will fix!

**Comment by dnil** [comment]
@jemten will have a look at the MIP code for producing and scoring: it is not necessarily fully aware of the list nature either. He will test with VEP first, and see how the CSQ annotation looks. That means this PR will go on hold until further notice..

**Comment by dnil** [comment]
In fact, I think I will close it and we can resume from it or start anew whenever.

**Comment by northwestwitch** [comment]
I guess it's always good to test with real data. The format of this spliceAI looks complicated!

**Pull request closed without merging**